src/main/java/com/petproject/pomodoro/controller/ExceptionHandlingController.java

```diff
@@ -18,7 +18,7 @@
                 .build();
     }
 
-    @ExceptionHandler(NoSuchElementException.class)
+    @ExceptionHandler(UserNotFoundException.class)
     public ResponseEntity<Void> handleNoSuchElementException() {
```

Comment:
Don't forget to change the name of methods)

Reply:
My bad)